digits/model/tasks/caffe_train.py

```diff
@@ -798,15 +806,42 @@ def task_arguments(self, resources, env):
                 identifiers.append(identifier)
             if len(identifiers) == 1:
                 args.append('--gpu=%s' % identifiers[0])
+                if win_python_layer:
+                    win_python_layer_gpu_id = identifiers[0]
```

Comment:
This is one GPU case.  Line 817~818 throws exception and the browser will show it.  Like this:
![image](https://cloud.githubusercontent.com/assets/5758988/15724965/69703efc-27fe-11e6-8cb9-00d7d41ebbb4.png)